docs/source/configuration/qos_configs/qos_client_config.rst

```diff
@@ -0,0 +1,53 @@
+.. _qos-client-configuration:
+
+QoS Client Configuration
+========================
+
+.. warning::
+
+    If you are using Cassandra, then you can use the AtlasDB QoS to rate-limit requests to Cassandra. Note that this
+    service is under active development and testing, please contact the AtlasDB team before using this feature.
+
+
+You will need to update your AtlasDB configuration in order to have said clients limit the read and write requests to
+Cassandra.
+
+.. note::
+
+    QoS client configuration is a part of the AtlasDB Runtime configuration.
+
+Install-Time Configuration
+--------------------------
+
+An AtlasDB config can have an optional ``qos`` block if the service wants to rate limit the reads/writes to Cassandra. This is
+live-reloadable and hence the limits can be modified while the service is online. The limits will be loaded when the
+next request hits the server and knowledge of the previous rate-limiting will be lost.
+
+Optional parameters:
+
+.. list-table::
+    :widths: 5 40
+    :header-rows: 1
+
+    *    - Property
+         - Description
+
+    *    - maxBackoffSleepTime (HumanReadableDuration)
```

Comment:
Maybe unnecessary to say it's a `HumanReadableDuration ` twice. But OK if you're worried people will miss it.